pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -204,18 +204,23 @@ def on_load_checkpoint(self, callback_state: Dict[str, Any]):
         self.best_model_score = callback_state["best_model_score"]
         self.best_model_path = callback_state["best_model_path"]
 
-    def save_checkpoint(self, trainer, pl_module):
+    def save_checkpoint(self, trainer, unused: Optional = None):
         """
         Performs the main logic around saving a checkpoint.
         This method runs on all ranks, it is the responsibility of `self.save_function`
         to handle correct behaviour in distributed training, i.e., saving only on rank 0.
         """
+        if unused is not None:
```

Comment:
See my above comment.
```suggestion
        if unused is not pl_module:
```

Reply:
My above code suggestion was wrong... The correct one is:
```suggestion
        if pl_module is not None:
```

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/6136#discussion_r587418452